frontend/src/components/project/ProjectCommentsContent.js

```diff
@@ -26,7 +26,22 @@ export default function CommentsContent({ user, project, token, setCurComments }
   const comments = project.comments;
 
   const handleRemoveComment = (c) => {
```

Comment:
Same here, use readable argument name here.

Reply:
updated variable names to something more descriptive.